lib/io/sqlite/setup.js

```diff
@@ -0,0 +1,55 @@
+import {
+  dbFilePath,
+  dbConfigExists,
+  writeDatabaseJSON,
+  auroraDirContext
+} from "../util.js";
+const ENVs = ["test", "prod", "dev"];
+
+function createDatabaseConfig() {
+  if (!dbConfigExists()) {
```

Comment:
Nitpick: The guard pattern is a little nicer here.

For example:

```
if (dbConfigExists()) return;

// Do the rest of the things 
```

Reply:
Done.